lib/platform.py

```diff
@@ -143,6 +143,12 @@ def info():
   else:
     os='linux'
 
+  try:
+    import subprocess
+    memory = int(float(subprocess.getoutput("free -h|awk '/Mem/ {print $2;}'").replace('Gi', '')) + 0.5)
```

Comment:
It would be better to use python modules that already contain this information...
For example, `os.sysconf` can be used like here: https://github.com/intel/vaapi-fits/blob/2f07b41ed9ca12beffa5ca7f9de394da009f7bbd/lib/metrics2/util.py#L46-L47
...or the `psutil` library can be used.